src/codelabs/async-await.md

```diff
@@ -217,7 +217,7 @@ Now that you have an `async` function, you can use the `await` keyword to wait
 for a future to complete:
 
 ```dart
-print(await createOrderMessage());
+print(await printOrderMessage());
```

Comment:
This instance of `createOrderMessage()` should not have been renamed. It doesn't make sense to have `print(await printFoo())`. As @natebosch pointed out, the rename should have been applied only to those definitions of `createOrderMessage()` that printed the order message in their body -- @natebosch [wrote](https://github.com/dart-lang/site-www/issues/2128#issuecomment-558307422) (emphasis mine):

> +1 for changing the name between `create` and `print`
> **depending on the behavior, return vs print**.

As I mentioned in point (2) of https://github.com/dart-lang/site-www/issues/2128#issue-526758837, the function name `createOrderMessage` was being used for *two different purposes*:
1. In one case to name a function that _returns an order message_, and 
2. In the other case to name a function that _prints an order message_.

My suggestion (and as I understand it, @natebosch's suggestion as well) was to rename to `printOrderMessage` _only_ those occurrences of `createOrderMessage()` that printed the message and returned nothing.

Reply:
ah, yes. Sorry I missed this during my review.

Reply:
@chalin Would you be amenable to leaving `printOrderMessage()` in, and removing the `print()` method that's wrapping it?

Reply:
Sure, although I thought that part of the goal was to show a function with a non-`void` return type evolve from a(n incorrect) use of `T` to `Future<T>`.

In the end, it's your call, but if you do opt to making the change that you propose, then the codelab's text should be carefully reviewed to ensure that it still makes sense with the updated code (e.g. to avoid things like https://github.com/dart-lang/site-www/pull/2137/files#r351422017).

IMHO, it would be easier to restore the original `createOrderMessage()` function name (for those places where the body actually returns a string).

Reply:
I'm curious to get your thoughts on [the following example](https://gist.github.com/25ade03f0632878a9169209e3cd7bef2): 
![Screen Shot 2019-11-27 at 9 47 17 AM](https://user-images.githubusercontent.com/5550175/69748048-61647d00-10fc-11ea-8f8b-b5af54e4cd72.png)

The function both prints and "creates" an order message (uses the result from fetching the user order) but doesn't explicitly return. Do you have an opinion about the name we should use?

Reply:
Would go with naming the first function in that example as `printOrderMessage()`.